desktop/components/commercial_filter/filters/keyword/index.jade

```diff
@@ -1,13 +1,14 @@
 #keyword-search-bar-container.typeahead
   input#keyword-search-bar-input.bordered-input(
-    placeholder='Search…'
+    placeholder='Filter by style, subject matter, title, artist, gallery, etc.'
     tabindex='1'
     itemprop='query-input'
     type='text'
     name='keyword'
-    value= keyword
```

Comment:
I think I added this to support pre-population of the search box from a GET variable. Would be nice to continue supporting that.

Reply:
Although obviously if it clashes with an informative placeholder text then we can leave it.

Reply:
We actually don't need it! We have client-side URL parsing and pillbox/filter support. So linking to a `?keyword=..` page works fine. The search term becomes a pillbox and is applied (similar to other filters).

Reply:
Oh, great, much better then! :+1: